tests/compile/test_fusion_attn.py

```diff
@@ -170,11 +177,18 @@ class TestAttentionFp8StaticQuantPatternModel(AttentionQuantPatternModel):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self.fp8_linear = Fp8LinearOp(
-            act_quant_static=self.quant_key.scale.static,
-            act_quant_group_shape=self.quant_key.scale.group_shape,
+        if self.quant_key.scale.group_shape.is_per_tensor():
+            weight_quant_strategy = ScaledMMLinearQuantStrategy.TENSOR
+        else:
+            weight_quant_strategy = ScaledMMLinearQuantStrategy.CHANNEL
+
+        self.fp8_linear = init_fp8_linear_kernel(
+            act_q_static=self.quant_key.scale.static,
+            act_q_group_shape=self.quant_key.scale.group_shape,
+            weight_quant_strategy=weight_quant_strategy,
+            out_dtype=torch.get_default_dtype(),
+            module_name=self.__class__.__name__,
```

Comment:
Can you add a utility that can be reused across fusion tests that constructs the layer, passes it to the kernel, etc.